_episodes/02-starting-with-data.md

```diff
@@ -246,9 +247,12 @@ of data:
 ~~~
 {: .output}
 
-Never fear, all the data is there, if you scroll up. Selecting just a few rows, so it is
-easier to fit on one window, you can see that pandas has neatly formatted the data to fit
-our screen:
+Don't worry: all the data is there! You can confirm this by scrolling upwards, or by
+looking at the `[# of rows x # of columns]` block at the end of the output.
+
+If you use `.head()` to view only a subset of rows, you will observe an output
```

Comment:
This is our first encounter of `.head()` so... instead of saying `If you use`,  let's declare that we can use `surveys_db.head()` to view only the first few rows of our dataset. I also think we should use variable name here as learners might think that `.head()` is a function on its own and try using it as `.head(something)`